frontend/src/pages/Resources/Resources.js

```diff
@@ -34,9 +40,31 @@ export default function Resources() {
             <section className="main-content">
               <section className="resource-list">
                 <SearchForm />
+                <div className="search-suggestions">
+                  <h4 class="header-description">Filters:</h4>
+                  {filters.map((filter, filterIdx) => {
+                    return (
+                      <p
+                        key={filterIdx}
+                        className={`search-suggestion ${resourceFilter === filter ? "active" : ""}`}
+                        onClick={() => handleClickFilterChip(filter)}
+                      >
+                        {filter.charAt(0).toUpperCase() + filter.slice(1)}
+                      </p>
+                    );
+                  })}
+                </div>
                 {renderedResources && renderedResources.length ? (
                   <div className="resources-list">
-                    <ResourceList resources={renderedResources} />
+                    {resourceFilter === "all" ? (
+                      <ResourceList resources={renderedResources} />
+                    ) : (
+                      <ResourceList
+                        resources={renderedResources.filter(
+                          (r) => r.type === resourceFilter
+                        )}
+                      />
+                    )}
```

Comment:
NIcely done :)

- can you move the filtering logic to a variable at the top and just pass that variable here. Maybe a name like `filteredResources` could work.
- while on the filtering logic, rename `r` to a descriptive varible name, `resource`/`renderedResource` should work fine :)